File: src/main/java/no/nav/syfo/controller/domain/adresse/UstrukturertAdresse.java
package no.nav.syfo.controller.domain.adresse;

import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;
import java.io.Serializable;

@Data
@Accessors(fluent = true)
@EqualsAndHashCode
public class UstrukturertAdresse implements Serializable {
    public String adresselinje1;
    public String adresselinje2;
    public String adresselinje3;
    public String adresselinje4;
    public String landkode;
}

File: src/main/kotlin/no/nav/syfo/controller/domain/adresse/UstrukturertAdresse.kt
package no.nav.syfo.controller.domain.adresse

import java.io.Serializable

data class UstrukturertAdresse(
    var adresselinje1: String? = null,
    var adresselinje2: String? = null,
    var adresselinje3: String? = null,
    var adresselinje4: String? = null,
    var landkode: String? = null
) : Serializable
